.scripts/update-gh-pages

```diff
@@ -36,24 +27,18 @@ else
     git clone --branch gh-pages "$REPO" "$CHECKOUT_PATH"
 fi
 
-if [ "$TRAVIS_BRANCH" != "" ]; then
-    echo " ==> Install: jazzy"
-    gem install --no-rdoc --no-ri jazzy
```

Comment:
Sorry if that's a dumb question: How's the binary dependency resolved now if we skip the installation step?

Reply:
Not dumb. Jazzy is pre-installed as per https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md